### PR TITLE
[Ready to be Merged] Create Physical Backed Token Module

### PR DESCRIPTION
**Pull request by darkty0x**

### CODE REVIEW

File: PhysicalBackedToken/contracts/interface/IBunzz.sol [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title IBunzz
+ */
+interface IBunzz {
```
Comment (Top-Kraken) [inline PhysicalBackedToken/contracts/interface/IBunzz.sol:7]: There is no usage of IBunzz interface. Please remove this.

File: PhysicalBackedToken/package.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    "prettier": "prettier --config .prettierrc --write \"{contracts,src}/**/*.{js,json,sol,ts}\"",
+    "prettier:contracts": "prettier --config .prettierrc --list-different \"{contracts,src}/**/*.sol\""
+  },
+  "devDependencies": {
+    "@nomiclabs/hardhat-ethers": "^2.0.5",
+    "@nomiclabs/hardhat-etherscan": "^3.0.3",
+    "@nomiclabs/hardhat-truffle5": "^2.0.7",
+    "@nomiclabs/hardhat-waffle": "^2.0.3",
+    "@nomiclabs/hardhat-web3": "^2.0.0",
+    "@openzeppelin/contracts": "^4.8.0",
```
Comment (Top-Kraken) [inline PhysicalBackedToken/package.json:26]: As you know, in Bunzz, we can only use 4.3.2 version for OpenZeppelin.

**Review by Top-Kraken — CHANGES_REQUESTED** [review]
And there is no test scripts for the project. We need test scripts for the usage of PBT and PBTRandom.

**Comment by solcmoon** [comment]
> 

I agree.
@moco-bunzz you need to write test script.
Please check https://lastrust.notion.site/Checklist-for-coding-reviewing-phase-d8c89b8eee0f4656ba4719488d9fbbfa

**Review by ghost — APPROVED** [review]

**Review by Top-Kraken — APPROVED** [review]

**Review by Top-Kraken — APPROVED** [review]

**Comment by zkitty-norika** [comment]
Hi @moco-bunzz, thank you for assigning me.
Priority-wise, preparing the audit report is not the first task according to the discussion with GK and Shota.
Now I'm focusing on the improvement of documentation and code quality.
I'd love to review from that point of view.

**Pull request closed without merging**